README.md

```diff
@@ -1852,10 +1852,10 @@ We grant `GameplayAbilities` on the server which then automatically replicates t
 
 The Sample Project stores a `TArray<TSubclassOf<UGDGameplayAbility>>` on the `Character` class that it reads from and grants when the game starts:
 ```c++
-void AGDCharacterBase::AddCharacterAbilities()
+void AGSCharacterBase::AddCharacterAbilities()
```

Comment:
Should still be AGDCharacterBase

Reply:
Sorry I missed this one, should be fixed in the last commit.